content/dstu2/authorization.md

```diff
@@ -2,195 +2,1061 @@
+your client application must first register using our 
+[code console][CERNER-CODE-CONSOLE]. A Cerner Care account is required to 
+register applications. Obtaining a Cerner Care account is free and 
+requires nothing but an email address.
 
-* state - to prevent [cross-site request forgery attacks][2]
-* redirect_uri - Note: The redirect_uri **must** match what was originally registered and requested
+Once registered, a **client identifier** will be provided for use with 
+the Cerner Authorization Server. As a registered client, Cerner 
+organizations may then ask for your client application to be enabled, 
+which is necessary in order to gain access to their protected resources.
 
-The authorization server authorize URL can be found in the security section of the [conformance document](../conformance).
+## Requesting Authorization on Behalf of a User ##
 
-If the user does not currently have an active session, the Authorization Server will redirect the user to the identity provider for the client organization in order to authenticate.
+To access Cerner's FHIR<sup>®</sup> services on behalf of a user, 
+your client application must make an authorization request through
+a user agent on the user's device.  This involves the following
+steps:
 
-If successful, the authorization server will redirect the user-agent back to your client applications redirect_uri with a **code** query parameter. This is the authorization code that can be exchanged for an access token.
+- Discover the authorization and token endpoint URLs.
+- Construct an authorization grant request URL.
+- Invoking a browser with the authorization grant request URL.
+- Receive the OAuth callback.
+- Exchange the received grant for a token.
+- Close the browser.
+- Handle any exceptions encountered.
+- Utilizing "refresh" tokens.
 
-```
-https://test.com/cb?code=1234-567890ab-cdef
-```
+A formal technical specification for this process may be found
+in the 
+[Cerner FHIR<sup>®</sup> Service Authorization Specification][AUTHZ-SPEC].
 
-Note: Using embedded browsers prevents single sign-on and authentication may fail for certain organizations who implement third party authentication systems.
+### Discovering Authorization URLs ###
+Each instance of Cerner's FHIR<sup>®</sup> services advertise 
+the URL of itsrespective authorization server within its 
+FHIR<sup>®</sup> conformance document.
 
-### Requesting an access token ###
-Once an authorization code has been acquired, a back-channel **POST** to the authorization server's token endpoint can be made to request an access token. Following the [OAuth 2.0][1] spec, the authorization server accepts a content type of **application/x-www-form-urlencoded** from your client application with the following information.
+The FHIR® Conformance resource can be retrieved by performing
+an HTTP GET against the resource located at the relative path
+of ``../Conformance`` from the FHIR<sup>®</sup> base URL.
```

Comment:
This makes it sound like the Conformance endpoint is "https://fhir-ehr.sandboxcerner.com/dstu2/Conformance" rather than "https://fhir-ehr.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/metadata".  May want to replace "../Conformance" with "./metadata"